tasks/runtime.yml

```diff
@@ -43,6 +43,11 @@
   notify:
     - Restart Munge service
 
+- name: Set fact for login-only nodes
```

Comment:
Just to note that ansible provides `group_by` which does something very similar to what you want to achieve:

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/group_by_module.html

Reply:
I did think of that. Seemed like you'd just end up with 2x groups (login_only, not_login_only) which would be less clear?